From e17161537f57dfa8a90f3f5418225e08607f8ad8 Mon Sep 17 00:00:00 2001
From: AltArch Kernel <noreply@centos.org>
Date: Thu, 24 Sep 2020 16:05:06 +0800
Subject: [PATCH 1/3] ACPICA: ACPI 6.3: PPTT add additional fields in Processor
 Structure Flags

mainline inclusion
from mainline-5.1-rc1
commit b5eab512e7cf
category: feature
bugzilla: 16072
CVE: NA
---------------------------

ACPICA commit c736ea34add19a3a07e0e398711847cd6b95affd
---
 include/acpi/actbl1.h | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/include/acpi/actbl1.h b/include/acpi/actbl1.h
index 531e527..a2c4989 100644
--- a/include/acpi/actbl1.h
+++ b/include/acpi/actbl1.h
@@ -1313,8 +1313,11 @@ struct acpi_pptt_processor {
 
 /* Flags */
 
-#define ACPI_PPTT_PHYSICAL_PACKAGE          (1)	/* Physical package */
-#define ACPI_PPTT_ACPI_PROCESSOR_ID_VALID   (2)	/* ACPI Processor ID valid */
+#define ACPI_PPTT_PHYSICAL_PACKAGE          (1)
+#define ACPI_PPTT_ACPI_PROCESSOR_ID_VALID   (1<<1)
+#define ACPI_PPTT_ACPI_PROCESSOR_IS_THREAD  (1<<2)     /* ACPI 6.3 */
+#define ACPI_PPTT_ACPI_LEAF_NODE            (1<<3)     /* ACPI 6.3 */
+#define ACPI_PPTT_ACPI_IDENTICAL            (1<<4)     /* ACPI 6.3 */
 
 /* 1: Cache Type Structure */
 
-- 
1.8.3.1

